boards/st/nucleo_h7s3l8/nucleo_h7s3l8.dts

```diff
@@ -166,6 +167,14 @@
 	clock-frequency = <I2C_BITRATE_FAST>;
 };
 
+&fdcan1 {
+	pinctrl-0 = <&fdcan1_rx_pd0 &fdcan1_tx_pd1>;
+	pinctrl-names = "default";
+	clocks = <&rcc STM32_CLOCK(APB1_2, 8)>,
+		<&rcc STM32_SRC_HSE FDCAN_SEL(0)>;
```

Comment:
Non blocking: Adding `clocks` here is optional, but it could be clearer to keep it indeed.